yum.repos.d/epel.repo

```diff
@@ -0,0 +1,26 @@
+[epel]
+name=Extra Packages for Enterprise Linux 5 - $basearch
+baseurl=http://mirrors.kernel.org/fedora-buffet/archive/epel/5/$basearch
+#mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-5&arch=$basearch
+failovermethod=priority
+enabled=1
```

Comment:
Enabling epel repo make this Docker image behaving differently than original `centos:5.11`, so I cannot accept this repo file as is.
At least we should disable this repo by default, to keep original behavior.

```suggestion
enabled=0
```